olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -395,6 +429,13 @@ DefaultCache::StorageOpenResult DefaultCacheImpl::SetupStorage() {
       settings_.disk_path_mutable = boost::none;
       result = DefaultCache::OpenDiskPathFailure;
     }
+
+    if (mutable_cache_) {
+      auto it = mutable_cache_->NewIterator(leveldb::ReadOptions());
+      for (it->SeekToFirst(); it->Valid(); it->Next()) {
+        mutable_cache_data_size_ += it->key().size() + it->value().size();
+      }
+    }
```

Comment:
Any idea how much this will take? Is there any increase in opening time with this?